### DISCUSSION TITLE
Support for routing - FHIR Communication or Messaging in AU Core for system to system information exchange

### DISCUSSION THREAD

**Discussion by dattachandan** [hl7au/au-fhir-core#49] [Q&A]
Hi team

Is there any plan for incorporating the FHIR [Communication](https://build.fhir.org/communication.html) or[ FHIR Messaging](https://hl7.org/fhir/R4/messageheader.html) in AU Core? Thanks for the P1 list
https://build.fhir.org/exchanging.html#message

![image](https://github.com/hl7au/au-fhir-core/assets/2154248/be724e11-9a75-425d-95dd-9937f63e8954)

![image](https://github.com/hl7au/au-fhir-core/assets/2154248/e7d15b9e-bd93-4837-b9c9-f6ba483c6ede)

**Comment by dt-r** [comment]
Hi @dattachandan, what would be your use for FHIR Messaging and/or implementation of the Communication pattern in AU Core?

**Reply by dattachandan** [reply]
Could be multiple system to system communication or service requests while a FHIR system needs to communicate to non-FHIR systems or other FHIR systems using both FHIR JSON or non-FHIR JSON, triggered by request for information or service by users of the system

**Comment by dt-r** [comment]
@dattachandan, I've moved this into our discussions area under Q&A.
